src/xcode/ENA/ENA/Source/Client/Security/iOS 12 Fallback/CryptoKitFallbacks.swift

```diff
@@ -346,3 +346,68 @@ extension String {
 		}
 	}
 }
+
+
+// MARK: - Digital Green Certificate
+
+struct DGCRSAKeypair {
```

Comment:
I personally would not label this 'DGC…' and stick to a general `RSAKeyPair`. Same for `publicKeyForBackend`. Maybe even add the key length as optional parameter in the initializer with our 3072bits as default.